src/lj_api.c

```diff
@@ -578,6 +580,20 @@ LUA_API size_t lua_objlen(lua_State *L, int idx)
   }
 }
 
+LUA_API size_t lua_rawlen(lua_State *L, int idx)
```

Comment:
Hmm, this implementation is slightly different from lua_rawlen but that is due to Mike's 'enhancement' to the lua_objlen where he formats numbers into their string format and returns the resultant length.  I'm OK with letting this diverge.